nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -559,6 +559,11 @@
       replacements_plain:
         '__version__ = get_distribution(__name__).version': 'raise DistributionNotFound'
 
+- module-name: 'cmsis_pack_manager'
+  implicit-imports:
+    - depends:
+        - 'cmsis_pack_manager.cmsis_pack_manager'
```

Comment:
In that module code there is 

> from .cmsis_pack_manager

That ought to mean it's not necessary to have this.

Reply:
OK